ui.apps/src/main/content/jcr_root/etc/clientlibs/fd/themes/hclecomm/hclecomm-theme/css/theme.css

```diff
@@ -0,0 +1,1881 @@
+.guideImage img{
```

Comment:
@rakeshkumar-verma : what is the source of the css code ?

Reply:
@rishabh-behl  this is default generated css, few things i have updated like font- family.